stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -638,6 +648,30 @@ public static void main(String[] args) {
     cli(args, Starter.class);
   }
 
+  private class BundleFailureListener implements FrameworkListener {
+
+    @Override
+    public void frameworkEvent(FrameworkEvent event) {
+      Throwable throwable = event.getThrowable();
```

Comment:
What happens if we wrap and throw a `RuntimeException` in here? If it causes the whole process to die that might less complex.

Reply:
Die in a nice way :)

Reply:
It looks like OSGi just swallows whatever is thrown from listeners.